templates/navigation-bar-new.twig

```diff
@@ -146,6 +150,27 @@
 			</span>
 		</button>
 	</div>
+	{% if mobile_tabs_menu %}
+	<div id="nav-mobile">
+		<nav id="nav-mobile-menu">
+			<ul>
+				{% for key,item in navbar_menu_items %}
+				<li class="nav-item {{ item.class }} {{ item == item.current ? 'active' : '' }}">
+					{% set link_ga_action = item.title %}
```

Comment:
I see that [in the actual navbar](https://github.com/greenpeace/planet4-master-theme/blob/829716999de29a001b93909880dd26e76f8b46f2/templates/navigation-bar-new.twig#L28-L31) we manually set "Act" and "Explore" for the `link_ga_action`, I'm guessing because of potential translations? Maybe we should do the same here? 🤔 Otherwise this variable is pretty useless here and I would just remove it.

Reply:
When we drop Act/Explore customization we will rely on item title to set the `link_ga_action` even on the main navbar. It's part of [PLANET-6497[(https://jira.greenpeace.org/browse/PLANET-6497). Since this is hidden behind a feature flag, I think it's fine to have this here already.